.github/scripts/libjpeg-turbo/build.sh

```diff
@@ -25,7 +25,6 @@ echo "::endgroup::"
 if [ $RUN_INSTALL = 1 ] ; then
     echo "::group::Install libjpeg-turbo"
         cmake --install $LIBJPEG_TURBO_BUILD_PATH
-
```

Comment:
Not sure this change is needed

Reply:
This is just codestyle fix to make this script formatting consistent with `.github/scripts/libxml2/build.sh`